include/ola/strings/Utils.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * Utils.h
+ * Miscellaneous string functions.
+ * Copyright (C) 2014 Simon Newton
+ */
+
+/**
+ * @file ola/strings/Utils.h
+ * @brief Miscellaneous string functions
+ */
+
+#ifndef INCLUDE_OLA_STRINGS_UTILS_H_
+#define INCLUDE_OLA_STRINGS_UTILS_H_
+
+#include <stdint.h>
+#include <iomanip>
+#include <iostream>
+#include <limits>
+#include <ostream>
+#include <sstream>
+#include <string>
+
+namespace ola {
+namespace strings {
+
+/**
+ * @brief Copy a string to a fixed length buffer without NULL terminating.
+ * @param input The string to copy to the buffer.
+ * @param buffer The memory location to copy the contents of the string to.
+ * @param size The size of the memory buffer.
+ * @note This the buffer may not be NULL terminated, it's not safe to use
```

Comment:
Is it worth a note saying it should be used for RDM and similar, for normal string copying use strncpy?